app/src/main/java/org/mozilla/rocket/theme/ThemeSettingDialogBuilder.kt

```diff
@@ -29,6 +29,9 @@ class ThemeSettingDialogBuilder(
 
         return AlertDialog.Builder(activity, R.style.ThemeSettingDialog)
             .setView(customContentView)
+            .setOnDismissListener {
+                homeViewModel.onExitThemeSetting()
```

Comment:
I'll also consider to pass callback into this class to handle the events

Reply:
Per discussed, no need to change this one